takeshape/penny
# Implement product page in nextjs project

### Test Plan (Steps to test):

1. Go to the 3 product pages
1. Ensure they work, as much as they can right now. The t-shirt is most fleshed out. Ensure you can make selections and add-to-cart.
